src/file.ts

```diff
@@ -3551,7 +3552,7 @@ class File extends ServiceObject<File> {
    */
   save(
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
We can get rid of `// eslint-disable-next-line @typescript-eslint/no-explicit-any`.